gossip/src/crds.rs

```diff
@@ -309,7 +323,10 @@ impl Crds {
                     Err(CrdsError::InsertFailed)
                 }
             }
-        }
+        };
+        //notify geyzer interface
+        self.notify_clusterinfo_update(self.table.get(&gossip_label));
```

Comment:
What about for the upsert case?

Reply:
Sorry, I didn't understand. It's clusterinfo you prefer cluster_info?